cfme/common/__init__.py

```diff
@@ -286,10 +286,12 @@ def get_tags(self, tenant="My Company Tags"):
             List of tags in format "Tag_category: Tag_name"
         """
         view = navigate_to(self, 'Details')
-        tags = view.tag.read()
+        tags = view.smart_management.read()
```

Comment:
Here ```smart_management = SummaryTable("Smart Management")```   of ```DetailsAnsibleCatalogItemView``` can be used to retrieve smart_mangt table info. By replacing ```view.tag.read()``` to ```view.smart_management.read()``` can reduce duplication of code which possibly would happen for new ```tag = SummaryTable("Smart Management")``` widget.